Samples/Weather.cs

```diff
@@ -19,28 +19,118 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+using System.Runtime.InteropServices;
 
 namespace OpenZiti.Samples {
+
     public class Weather {
         static MemoryStream ms = new MemoryStream(2 << 16); //a big bucket to hold bytes to display contiguously at the end of the program
+        static ZitiTunnelCommand.Options tunOptions = new ZitiTunnelCommand.Options();

```

Comment:
extract to an MFA sample and i'll review again. keep the existing samples intact